charts/uws-api-server/values-nts-dev.yaml

```diff
@@ -0,0 +1,41 @@
+targetCluster: "nts"
+hostname: lsst-nts-k8s.ncsa.illinois.edu
+basePath: "dev-uws-server"
+
+workingVolume:
+  name: job-files
+  server: lsst-l1-cl-arctl.ncsa.illinois.edu
+  exportPath: "/scratch/uws"
+  claimName: uws-dev-server-pvc
```

Comment:
Maybe `dev-uws-server-pvc` for parallelism with the `basePath` and the other volume claims below?  I believe this name is arbitrary.

Reply:
Done